microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Share performance and memory traces
+description: Learn to export and import performance and memory traces in DevTools to get help when investigating issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/06/2023
+---
+# Share performance and memory traces
+
+The **Performance** and **Memory** tools record runtime data about your webpage. Exploring the recorded data makes it possible to improve your webpage's memory usage or runtime performance.
+
+Additionally, the recorded data can be exported to files on disk. The exported files are called _traces_.
+
+Exporting performance and memory traces is useful when you want to share these files with other people to get help with investigating issues. An exported trace file can be imported in DevTools at any time.
+
+Traces can be exported in one of two ways: 
+
+* As a normal trace: a `.json`, `.heapsnapshot`, `.heapprofile`, or `.heaptimeline` file, which contains minimal information about the runtime data from the traced webpage.
```

Comment:
consistency: here, "sourcemaps" 1 word

Reply:
I re-structured the article so it's more of a description of this new experimental feature, with its value prop, rather than a description of the 2 ways to export.